themes/custom/droopler_theme/scss/layout/_modifiers.scss

```diff
@@ -451,8 +451,11 @@ $d-p-themes: map-merge(
 
 @each $name, $themeSettings in $d-p-themes {
   .theme-#{$name} {
-    > .paragraph,
-    > div[data-geysir-paragraph-id] > .paragraph,
+    > .paragraph:not(.content-moved-inside),
+    > div[data-geysir-paragraph-id] > .paragraph:not(.content-moved-inside),
+    &.theme-custom > .content-moved-inside,
+    &.theme-custom > .content-moved-inside > *:not(.content-inside-wrapper),
```

Comment:
tutaj to potrzebne, żeby poprawnie dziedziczyło customowe kolorki w poszczególnych divach.. 
chociaż ta druga linijka, to tak naprawdę potrzebna było chyba tylko dla paragrafu side embed, dla wariantu:
![image](https://user-images.githubusercontent.com/37533838/102213868-ad38be80-3ed7-11eb-8275-9510102cc190.png)